.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -0,0 +1,65 @@
+const {exec} = require('child_process');
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+// Use Github Actions' default environment variables to get repo information
+// https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
+const [repoOwner, repoName] = process.env.GITHUB_REPOSITORY.split('/');
+
+const MAX_RETRIES = 10;
+let errCount = 0;
+let shouldRetry;
+
+do {
+    shouldRetry = false;
+    exec('npm version prerelease -m "Update version to %s"', (err, stdout, stderr) => {
+        console.log(stdout);
+        if (err) {
+            console.log(stderr);
+
+            // It is possible that two PRs were merged in rapid succession.
+            // In this case, both PRs will attempt to update to the same npm version.
+            // This will cause the deploy to fail with an exit code 128, saying the git tag for that version already exists.
+            if (errCount < MAX_RETRIES) {
+                console.log(
+                    'Err: npm version conflict, attempting to automatically resolve',
+                    `retryCount: ${++errCount}`,
+                );
+                shouldRetry = true;
+                const currentPatchVersion = `v${require('../../../package.json').version.slice(0, -4)}`
+                console.log('Current patch version:', currentPatchVersion);
+
+                // Get the highest build version git tag from the repo
+                console.log('Fetching tags from github...');
+                const octokit = github.getOctokit(core.getInput('githubToken'));
+                octokit.repos.listTags({
+                    owner: repoOwner,
+                    repo: repoName,
+                })
+                    .then(response => {
+                        const tags = response.data.map(tag => tag.name);
+                        console.log('Tags: ', tags);
+                        const highestBuildNumber = Math.max(
+                            ...(tags
+                                .filter(tag => tag.startsWith(currentPatchVersion))
+                                .map(tag => tag.slice(-3))
```

Comment:
This line could eventually break when we pass build `999`. Should we change this to instead get  everything after `-` instead?

Reply:
Good idea. Unlikely that this will be an issue because pretty soon we'll be periodically bumping the patch version and resetting the build version to 0. Nonetheless will fix

Reply:
In that case, we need to change it for when we reset the build version, unless we are 0 padding them to length of 3 😂. Either way, good to fix this rather than have it come back to bite us later